test/lib/heuristics/dichotomious_test.rb

```diff
@@ -23,11 +23,6 @@ def test_dichotomious_approach
       vrp = TestHelper.load_vrp(self)
       vrp.resolution_dicho_algorithm_service_limit = 457 # There are 458 services in the instance. TODO: Remove it once the dicho contions are stabilized
 
-      if ENV['TRAVIS'] # Compansate for travis performance
```

Comment:
Do we need specifying durations on CI Github Action ?  We can use `ENV['CI']` in place of `ENV['TRAVIS']`

Reply:
Tests work as is right now so I don't think we need anything like that for now.